.factorypath

```diff
@@ -0,0 +1,75 @@
+<factorypath>
```

Comment:
this is for eclipse only I believe and could be done with m2eclipse plugin right - although not during import time..

Reply:
That's right, I thought I have removed this.